app/src/components/governance/PageProposal.vue

```diff
@@ -183,43 +59,50 @@
       <ModalVote
```

Comment:
Feels like overloading this component with the seconds. Maybe create another one?

Reply:
maybe in another PR?